lib/ansible/module_utils/facts/network/iscsi.py

```diff
@@ -62,8 +78,11 @@ def collect(self, module=None, collected_facts=None):
                     iscsi_facts['iscsi_iqn'] = line.split('=', 1)[1]
                     break
         elif sys.platform.startswith('aix'):
-            aixcmd = '/usr/sbin/lsattr -E -l iscsi0 | grep initiator_name'
-            aixret = subprocess.check_output(aixcmd, shell=True)
-            if aixret[0].isalpha():
-                iscsi_facts['iscsi_iqn'] = aixret.split()[1].rstrip()
+            rc, out, err = self.module.run_coomand('/usr/sbin/lsattr -E -l iscsi0 | grep initiator_name')
```

Comment:
I believe this should be `module.run_command()` since this class (nor its parents)does not have a `run_command` method.

This also assumes there is always a Module object. Maybe fall back to using `subprocess` if `module` is `None.`

Reply:
I see. Fixed as requested